pyfr/integrators/dual/phys/steppers.py

```diff
@@ -1,25 +1,139 @@
 # -*- coding: utf-8 -*-
 
+import math
+
 from pyfr.integrators.dual.phys.base import BaseDualIntegrator
 
 
 class BaseDualStepper(BaseDualIntegrator):
-    pass
 
+    def step(self, t, dt):
```

Comment:
Should we move this specific implementation to the DIRK subclass (at least for the moment)?

Reply:
Just following up here.